contrib/example-backend/controllers/clusterbinding/clusterbinding_controller.go

```diff
@@ -114,6 +176,45 @@ func (c *Controller) enqueueServiceExport(logger klog.Logger, obj interface{}) {
 	c.queue.Add(key)
 }
 
+func (c *Controller) enqueueServiceExport(logger klog.Logger, obj interface{}) {
+	seKey, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
```

Comment:
curiosity question: why not simply `MetaNamespaceKeyFunc(obj)` ?

Reply:
I want to get the key if the object was lost due to informer restart, i.e. obj is a tombstone